ground/gcs/src/plugins/uavtalk/telemetry.h

```diff
@@ -84,7 +84,9 @@ class Telemetry : public QObject
     Telemetry(UAVTalk *utalk, UAVObjectManager *objMngr);
     ~Telemetry();
     TelemetryStats getStats();
-    QByteArray *downloadFile(quint32 fileId, quint32 maxSize);
+    QByteArray *downloadFile(quint32 fileId, quint32 maxSize,
+            std::function<void(quint32)>progressCb = nullptr);
```

Comment:
Not really, it would be consistent with the rest of the codebase whereas this is not.